src/main/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/DoobiePetRepositoryInterpreter.scala

```diff
@@ -68,9 +74,14 @@ class DoobiePetRepositoryInterpreter[F[_]: Monad](val xa: Transactor[F])
     extends PetRepositoryAlgebra[F] {
   import PetSQL._
 
-  def put(pet: Pet): F[Pet] =
+  def create(pet: Pet): F[Pet] =
     insert(pet).withUniqueGeneratedKeys[Long]("ID").map(id => pet.copy(id = id.some)).transact(xa)
 
+  override def update(pet: Pet): F[Option[Pet]] = {
```

Comment:
remove the override keyword here

Reply:
There are a bunch of these in files not touched by this PR. I'll track them all down and amend in a separate PR if that's ok